### ISSUE TITLE
ISSN 2077-1444 | Religions | keine Rezensionen

### ISSUE THREAD

**Issue by IxKo** [ubtue/DatenProbleme#1777]
#### URL
https://www.mdpi.com/2077-1444/13/4/342/htm (IxTheo#2022-04-11#021F27857DBA9EAEC815DB6DADE5C6FA9975C866)
https://www.mdpi.com/2077-1444/13/4/299 (IxTheo#2022-03-30#AC8C0F0F1EB856E048E80F9857D087C6A4BC13C8)
https://www.mdpi.com/2077-1444/13/5/462 (IxTheo#2022-05-19#67B518B7AD258F884482AE2D77A63AD536BE954D)
  

### Problembeschreibung
Die Artikel wurden in die Test-DB als Rezensionen eingespielt. Es handelt sich jedoch um keine Rezensionen.

Im zotk-Verfahren mit ubtue_MDPI Journals.js werden sie nicht als Rezensionen gekennzeichnet.

**Comment by jriedl** [comment]
Nachtrag: Falls diese Zeitschrift grundsätzlich keine Rezensionen enthält, könnte man auch hier einfach über die Conf den `zotero_review_regex` auf einen nie matchenden Wert setzt. Das würde die Translatoranpassungen und vor allem die Seitenefekkte auf andere evtl. andere Zeitschriften, bei denen die Rezensionen tatsächlic über Review im Titel matchen, ersparen.

**Comment by helenanebel** [comment]
Liebe @IxKo ,
wenn wir das entfernen, würden gar keine Rezensionen mehr durch diesen Translator gekennzeichnet, die diese Kennzeichnung haben:
![grafik](https://user-images.githubusercontent.com/49190455/174332471-46d80716-4db5-4e39-b9d1-53b96d28bb72.png)
Wenn das so gewünscht ist, kann ich das umsetzen.
Viele Grüße, Helena Nebel

**Comment by jriedl** [comment]
> Gelten nicht trotzdem die GLOBAL JOURNAL SETTINGS für alle ZSs - selbst wenn für die ZS eine gesonderte RegEx benannt wird?

Ja, da habe ich mich geirrt, für die Rezensionen ist das tatsächlich so implementiert (meiner Ansicht nach sehr kontraintuitiv, aber man sollte jetzt wohl keine Seiteneffekte riskieren und das wieder ändern), d.h. der Ansatz mit dem `zotero_review_regex` kann leider nicht verwendet werden. Alternativ bietet sich `remove_marc_field_655a    = (?i)^Rezension$` an, der komplette Conf-Eintrag wäre also (immer unter der Voraussetzung, dass es sich bei allen "reviews" dieser Zeitschrift nie um echte Rezensionen handelt):

```ini
[Religions]
zeder_id                  = 1844
zeder_modified_time       = "2022-06-15 10:43:33"
zotero_update_window      = 46
online_ppn                = 665435797
online_issn               = 2077-1444
zotero_type               = RSS
zotero_group              = IxTheo
zotero_url                = https://www.mdpi.com/rss/journal/religions
zotero_expected_languages = eng
remove_marc_field_650a    = (?i)^Book\s+Reviews?$
remove_marc_field_655a    = (?i)^Rezension$
override_json_field_pages = "article %org%"
zotero_delivery_mode      = TEST
ssgn                      = FG_0
license                   = LF
selective_evaluation      = false
tiefp                     = N
zotero_subgroup           = RelBib
```


> Werden denn die Rezensionen überhaupt aufgrund des Titels als 1131 Rez. erkannt oder nicht durch etwas ganz anderes?

Ja, der Translator erkennt "review" im Titel und setzt dann als Keyword "Book Review". Daraus wird dann "Rezension" in 655.

**Comment by IxKo** [comment]
Rezensionen sollen nicht mehr über das Review-HTML-Tag auf dieser Plattform (mdpi) erkannt werden.
Dies ist die einzige ZS auf der Plattform und die ZS scheint in der Vergangenheit keinerlei echte Rezensionen gehabt zu haben.

**Comment by jriedl** [comment]
Übernahme des obigen Conf-Eintrags unterdrückt in diesem Fall Rezensionen und erspart Translator-Anpassungen.

**Comment by helenanebel** [comment]
Gut, dann verzichte ich auf eine Anpassung. Kann man dann auf Ready for testing setzen?

**Comment by jriedl** [comment]
Habe die eine Zeile in der Conf angepasst und eingechecked, sollte ab dem Pipelinedurchlauf heute Abend auf nu/ub28 sichtbar sein.

**Comment by IxKo** [comment]
Danke, es wird nichts mehr als Rezension gekennzeichnet.
